pymongo/database.py

```diff
@@ -926,7 +926,7 @@ def validate_collection(
         session: Optional["ClientSession"] = None,
         background: Optional[bool] = None,
         comment: Optional[Any] = None,
-    ) -> Dict[str, Any]:
+    ) -> _DocumentOut:
```

Comment:
This method always returns a dict so isn't the previous return type more correct?

Reply:
Same comment as above, it is the result of a `command()` call

Reply:
Used a cast here as well